cypress/integration/home_page_spec.js

```diff
@@ -8,6 +8,20 @@ describe('Not Fur-gotten Home Page User Flow', () => {
           'x-gql-operation-name': 'getAllPets',
         }
       },
+      {
```

Comment:
Nice work, also, with the order of these intercepts, so that the last one is loading the fixture with the additional pet and image link added to it. Great attention to detail.